pom.xml

```diff
@@ -53,6 +53,7 @@
              mvn validate -Pregen -N
         -->
         <camel-quarkus.version>1.0.0-M3</camel-quarkus.version>
+        <debezium-outbox-quarkus.version>1.1.0-SNAPSHOT</debezium-outbox-quarkus.version>
```

Comment:
No snapshots, please.

Reply:
That's fine, I can set this to `1.1.0.Final`; however this won't work until we release that artifact in a few weeks.

Reply:
The thing is that, we should be able to release this any time. So practically, everything here has to be of a "final" quality. Otherwise, we need to synchronize releases, which is counter productive.
Here is how it is supposed to work. As soon as any participant releases a final and the integration PR passes the testsuite, this thing can be released and consumed w/o waiting for others.
You should be able to test your snapshot and pre-final releases in your environment by pulling the latest master of this repo and setting your preferred version.
Would that work for you?

Reply:
That does, thanks.

Reply:
@aloubyansky what about Beta or CR releases, are those allowed?